addons/rose/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "build:development": "ember build",
     "build": "ember build --environment=production && cp metadata.json dist/metadata.json",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
```

Comment:
Non-blocking question for understanding: was script with just single quotes not working, or is this change related to the upgrade?

Reply:
I bet is the `ember-cli` during the update process who changed this, I saw it before when upgrading ember.

Reply:
yeah, it was automatically added and also checked it was included in the other addon ember upgrade PRs as well